src/client/models/MsgTransfer.ts

```diff
@@ -0,0 +1,26 @@
+import { PlainMessage } from "@bufbuild/protobuf";
```

Comment:
Let's name this file and function `MsgIbcTransfer` to make it more obvious what it does